File: examples/nodejs/src/version_update.ts
import { post } from './sock.js'

(async () => {
  const json = await post('/internal/version/update', {
    message: {
      "binary_url": "https://nilth.com/nodex-agent",
      "path": "/tmp/nodex-agent",
    }
  })

  console.log(json)
})()

File: examples/python/v2/win/version_update.py
from request import post


payload = {
    "message": {
        "binary_url": "https://example.com/nodex-agent-1.0.0.zip",
        "path": "C:\\Temp",
    }
}


json_response = post("/internal/version/update", payload)

print("The response is as follows.\n")
print(json_response)
